### DISCUSSION TITLE
RDR 2 Translation issue

### DISCUSSION THREAD

**Discussion by MrJimmerGaming** [Halen84/ScriptHookRDR2DotNet-V2#15] [Q&A]
Hi guys, i'm trying to translate the game to ukrainian language and looking for best option.
I tried to use "String Translator" mod and it works good excepts two things:
- This works for all languages, no matter which one is selected.
- For some reason, the mod cannot correctly replace some keys. Example: "IB_YES = Yes" used to confirm exiting the game, but replacing this key just makes the button disappear from the game.

I'm looking for YLDB converter and know that OpenIV can convert these files to .txt format. But i don't know how to convert it to .yldb again from my .txt file. Then I found this ScriptHookRDR2DotNet app and the question is: can i translate the game with it?

P.S. I also tried the Mod Activator with YLDB converter inside, but it fails everytime on any .yldb file.

I will be very glad for your help...

**Comment by Halen84** [comment]
ScriptHookRDR2DotNet is not used to change strings in the game, it's used to allow people to create C# scripts for the game. I'm pretty sure what you are describing can be done in general, but I don't know how to off the top of my head
